hw02-generics-collections/src/main/java/ru/otus/kovaleva/Customer.java

```diff
@@ -0,0 +1,55 @@
+package ru.otus.kovaleva;
+
+public class Customer {
+    private final long id;
+    private String name;
+    private long scores;
+
+    public Customer(long id, String name, long scores) {
+        this.id = id;
+        this.name = name;
+        this.scores = scores;
+    }
+
+    public long getId() {
+        return id;
+    }
+
+    public String getName() {
+        return name;
+    }
+
+    public void setName(String name) {
+        this.name = name;
+    }
+
+    public long getScores() {
+        return scores;
+    }
+
+    public void setScores(long scores) {
+        this.scores = scores;
+    }
+
+    @Override
+    public String toString() {
+        return "Customer{" + "id=" + id + ", name='" + name + '\'' + ", scores=" + scores + '}';
+    }
+
+    @Override
+    public boolean equals(Object o) {
+        if (this == o) return true;
+        if (o == null || getClass() != o.getClass()) return false;
+
+        Customer customer = (Customer) o;
+
+        if (id != customer.id) return false;
+        return true;
+    }
+
+    @Override
+    public int hashCode() {
+        int result = (int) (id ^ (id >>> 32));
+        return result;
```

Comment:
Есть готовый метод, можно его применить:
```suggestion
        return Long.hashCode(id);
```